src/Standards/Generic/Tests/CodeAnalysis/ForLoopShouldBeWhileLoopUnitTest.3.inc

```diff
@@ -0,0 +1,4 @@
+<?php
+
+// Intentional parse error. Testing that the sniff is *not* triggered in this case.
+for ($i = 0; $i < 10; $i++ {
```

Comment:
```suggestion
for ($i = 0; $i < 10; $i++
```

Reply:
I changed this in the original commit, as you asked me to amend the commits anyway.

Could you please elaborate on why you prefer the version without the opening bracket?

Reply:
As a rule of thumb, especially for code samples related to bugs, a _minimal_ code sample should be used which demonstrates the issue and safeguards the fix.

The `{` here is superfluous for the _minimal_ code sample.

In practice, keeping the opening curly would mean that every time I look at the code sample I will wonder whether the bug would not have been triggered without the curly and would need to check/confirm, which is time I could have spend on other things. (and I imagine that I'm not the only one whose brain works like this in the context of sniffs)

As an extra "helper" for people coming across this code, you could also consider expanding the inline comment a little, for example like this:
```diff
-// Intentional parse error. Testing that the sniff is *not* triggered in this case.
+// Issue PHPCSStandards/PHP_CodeSniffer#226
+// Intentional parse error (missing close parenthesis). Testing that the sniff is *not* triggered
+// in this case and that no PHP 8.3+ deprecation notice is thrown.
```

That makes the exact case you are testing explicit.

Reply:
Thanks for the additional information.

> As an extra "helper" for people coming across this code, you could also consider expanding the inline comment a little, for example like this:

I went ahead and expanded the inline comment.